tests/lib/cli-engine/cli-engine.js

```diff
@@ -4791,7 +4793,10 @@ describe("CLIEngine", () => {
 
             it("'executeOnFiles()' should not verify 'foo.js'.", () => {
                 const engine = new InMemoryCLIEngine();
-                const filePaths = engine.executeOnFiles("**/*.js").results.map(r => r.filePath);
+                const filePaths = engine.executeOnFiles("**/*.js")
+                    .results
+                    .map(r => r.filePath)
+                    .sort();
```

Comment:
Just so I understand, why is it possible for the order to have changed here?

Reply:
The reason for adding `.sort()` is https://github.com/eslint/eslint/runs/359919594#step:5:53 .

The `fs.readdir` of `memfs` looks like to iterate files in different orders between win32 and posix (those tests pass on my PC). I remembered that the `readdir` doesn't guarantee any ordering then I added `.sort()`.